### PR TITLE
DDCE-5519: Upgrade ui-test-runner

### PR DESCRIPTION
**Pull request by EdCope**
- Upgrade ui-test-runner to version 0.30.0

main: https://build.tax.service.gov.uk/job/DDCE%20Live%20Services/job/Advance%20Valuation%20Rulings/job/advance-valuation-rulings-ui-tests/68/

branch: https://build.tax.service.gov.uk/job/DDCE%20Live%20Services/job/Advance%20Valuation%20Rulings/job/advance-valuation-rulings-ui-tests/69/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,63 @@
...
+To start the required services via [service manager](https://github.com/hmrc/sm2), run:
 
-Run the following command to start services locally:
-
-```
-docker run --rm -d --name mongo -d -p 27017:27017 mongo:5.0
-```
-
-```
+```shell
```
Comment (frikit) [inline README.md:11]: change `shell` to `bash` -> bash is used as default in jenkins so locally we can try to have the same

File: build.sbt [hunk 2]
```diff
@@ -1,13 +1,12 @@
-ThisBuild / scalaVersion := "2.13.13"
+ThisBuild / scalaVersion := "2.13.14"
 
 lazy val testSuite = (project in file("."))
   .disablePlugins(JUnitXmlReportPlugin) // Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(
     name := "advance-valuation-rulings-ui-tests",
     version := "0.1.0",
```
Comment (frikit) [inline build.sbt:7]: our versions in ATs are all over the place, change this to 1.0.0 to align all of them, dont change anything in terms of build

File: build.sbt [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
     scalacOptions ++= Seq("-feature"),
```
Comment (frikit) [inline build.sbt:8]: dont need Seq for one param

**Review by frikit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Taiwo-AB) [reply README.md:11]: I would rather leave this one as shell and the other script ones as bash. This is what is used in the [sm2](https://github.com/hmrc/sm2/blob/main/README.md?plain=1#L77) readme. I have been updating mine to shell for the sm2 stuff.

**Review by frikit — APPROVED** [review]

**Pull request merged**